x-pack/test/api_integration/deployment_agnostic/apis/observability/alerting/burn_rate_rule.ts

```diff
@@ -60,31 +57,47 @@ export default function ({ getService }: FtrProviderContext) {
         indexing: { dataset: 'fake_hosts' as Dataset, eventsPerCycle: 1, interval: 10000 },
       };
       dataForgeIndices = await generate({ client: esClient, config: dataForgeConfig, logger });
-      await alertingApi.waitForDocumentInIndex({ indexName: DATA_VIEW, docCountTarget: 360 });
+      await alertingApi.waitForDocumentInIndex({
+        indexName: DATA_VIEW,
+        docCountTarget: 360,
+        roleAuthc,
+      });
       await dataViewApi.create({
+        roleAuthc,
         name: DATA_VIEW,
         id: DATA_VIEW_ID,
         title: DATA_VIEW,
       });
-      roleAuthc = await svlUserManager.createM2mApiKeyWithRoleScope('admin');
     });
 
     after(async () => {
-      await supertest.delete(`/api/alerting/rule/${ruleId}`).set(internalReqHeader);
-      await supertest.delete(`/api/actions/connector/${actionId}`).set(internalReqHeader);
+      await supertest
+        .delete(`/api/alerting/rule/${ruleId}`)
+        .set(roleAuthc.apiKeyHeader)
```

Comment:
`supertest` is initialized with `operator` user and its Cookie persist. Passing custom api key for specific role has no affect, API call will be executed with `operator` privileges.

If you test the APIs, it is critical to use `supertestWithoutAuth` only. If it is env cleanup and `admin` has no permissions to delete, using `supertest` is reasonable but not need to pass api key.